README.md

```diff
@@ -1,2 +1,37 @@
-# AR6-WG1-Data-Compilation
-Compilation of data from AR6 WG1
+# Data Compilation & Figures for the IPCC AR6 WG1
```

Comment:
```suggestion
# Data Compilation & Figures from the IPCC AR6 WG1
```

Might be clearer given we didn't make anything for IPCC, we're just using outputs